src/cpp/session/SessionConsoleProcess.cpp

```diff
@@ -993,7 +984,7 @@ void loadConsoleProcesses()
    }
    BOOST_FOREACH(const FilePath& child, children)
    {
-      // Don't erase the INDEX or any subfolders
+      // Don't erase the INDEXnnn or any subfolders
       if (!child.filename().compare(kConsoleIndex) || child.isDirectory())
```

Comment:
To consider: this effectively means that different versions of RStudio will destroy each others' console indexes (so you'll lose your console state on upgrade and downgrade). I don't think that's a big deal here (and especially not worth fixing for pre-release), but for versioned data we typically have a folder name with a monotonically increasing version number, and just have each RStudio version read/write from its own folder.